snippets-python/shared.py

```diff
@@ -35,15 +35,15 @@ def broadcast_transaction(transaction: Transaction):
     print(f"{EXPLORER_URL}/transactions/{hash}")
 
 
-def query_contract(address: Address, function: str, arguments: Sequence[Any]) -> Any:
-    query = ContractQueryBuilder(
+def query_contract(address: Address, function: str, arguments: list[Any]) -> SmartContractQueryResponse:
+    sc_controller = SmartContractController(chain_id=CHAIN_ID, network_provider=create_network_provider())
```

Comment:
Now, since it's much easier to write code based on the SDKs, maybe rewrite `adder.py` to use an entrypoint?

Furthermore, maybe remove `shared.py`?